helix-rest/src/main/java/org/apache/helix/rest/server/resources/zookeeper/ZooKeeperAccessor.java

```diff
@@ -52,12 +51,15 @@
   private static final Logger LOG = LoggerFactory.getLogger(ZooKeeperAccessor.class.getName());
   private BaseDataAccessor<byte[]> _zkBaseDataAccessor;
 
+  private static final String PATH_STR = "path";
+
   public enum ZooKeeperCommand {
     exists,
     getBinaryData,
     getStringData,
     getChildren,
-    getStat
+    getStat,
+    delete
```

Comment:
I think it would be smarter to use deleteEphemeral and rename your methods accordingly because it seems that it's  not the general delete you're trying to support.

Reply:
I don't think we want "deleteEphemeral" eventually. As mentioned in the description, this is a premature feature that we add now for unblocking our users.

Alternatively, I tried to allow deleting live instance only. But that will pollute the ZookeeperAccessor API with Helix logics. So I discarded that idea.

Reply:
I still don't agree - it seems that you are suggesting using `delete`, expecting its behavior to change in the future. That will bring about backward-compatibility issues and make the meaning of "delete" murky. 

It's fine if you need to provide another endpoint/API, but a good API design involves something that is 1) easy to do and 2) doing exactly what it's advertising to do. Do you see why it might be less desirable to add hidden assumptions to `delete`?

Reply:
I can see where you are coming from. Could you check the latest change that I have modified the method to use DELETE verb according to what Huizhi suggested? I think it is cleaner. However, in this case, we need some more parameters to separate the cases. And I think it might be overcomplicated.

@dasahcc and @pkuwm please also share your opinion since you also contributed to the Helix rest.

Reply:
My opinion is, 
1. use `DELETE` verb that is designed for the REST delete operation.
2. if we only want to support deleting ephemeral, document it well and return a clear response like:
```
    HTTP/1.1  404
    Content-Type: application/json
 
    {
      "message": "Deleting a non-ephemeral node is not supported/allowed",
      "path": "/a/b/c"
    }
```
And it is extensible if we want to support deleting persistent node in the future.

Reply:
This information is carried in the response entity as a string for now. I don't think we need to make it too structural (complicated) given it is a temporary restriction. And eventually, we do not have a clear standard for the response format now. So I would prefer holding on any more complex idea.